ibis/tests/expr/test_table.py

```diff
@@ -523,13 +523,14 @@ def test_groupby_alias(table):
     assert_equal(result, expected)
 
 
-def test_summary_expand_list(table):
-    summ = table.f.summary()
-
-    metric = table.g.group_concat().name('bar')
-    result = table.aggregate([metric, summ])
-    expected = table.aggregate([metric] + summ.exprs())
-    assert_equal(result, expected)
+# TODO(kszucs): we should perhaps change the API to
+# table.aggregate(metrics=table.f.summary()) ?
+# def test_summary_expand_list(table):
+#     summ = table.f.summary()
+#     metric = table.g.group_concat().name('bar')
+#     result = table.aggregate([metric, summ])
```

Comment:
It seems like this can be done by the `aggregate` method by something like this:

```python
exprs = []
for expr in inputs:
    op = expr.op()
    if isinstance(op, ops.Summary):
        exprs.append(op.exprs)
    else:
        exprs.append(expr)
```

Reply:
That could be the solution perhaps!

We don't yet have a `Summary` operation, we expose the summaries as a [list of expressions](https://github.com/ibis-project/ibis/blob/master/ibis/expr/api.py#L1232-L1239). 

Though that would mean that we shouldn't enforce an explicit [output type for all of the operations](https://github.com/ibis-project/ibis/blob/master/ibis/expr/operations.py#L156-L161).